src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -66,6 +73,9 @@ public void processNewAddressFromSourceId(
     Case newCaseFromSourceCase = buildCaseFromSourceCaseAndEvent(newCollectionCase, sourceCase);
 
     newCaseFromSourceCase = caseService.saveNewCaseAndStampCaseRef(newCaseFromSourceCase);
+    if (newCaseFromSourceCase.getUprn() == null) {
```

Comment:
I don't think this is correct. All new cases will have a dummy UPRN to distinguish them from sample cases